addons/stock/views/product_views.xml

```diff
@@ -36,6 +36,52 @@
             </field>
         </record>
 
+        <record id="action_product_replenishment" model="ir.actions.server">
+            <field name="name">Replenish</field>
+            <field name="groups_id" eval="[(4, ref('stock.group_stock_user'))]"/>
+            <field name="model_id" ref="product.model_product_product"/>
+            <field name="binding_model_id" ref="product.model_product_product"/>
+            <field name="binding_view_types">form</field>
+            <field name="state">code</field>
+            <field name="code">
+                for record in records:
```

Comment:
"Consumable Products" is now a technical term and not in the interface. Not a good explanation for user